esp/bindings/http/platform/httpsecurecontext.cpp

```diff
@@ -22,6 +22,7 @@ class CHttpSecureContext : extends CEspBaseSecureContext
 {
 private:
     CHttpRequest* m_request;
+    std::map<std::string, std::string> sessionData;
```

Comment:
A map makes sense if multiple manager-defined values are supported. A single manager-defined value (what's included in this PR) may be clearer as a two separate values (one for the flag and one for the manager data).

Reply:
I am using a map because we can add more values, like, in your comment above, a password.

Reply:
I know. My comment here was to reinforce the concern about naming a single value the ESP looks for versus multiple values whose names the ESP shouldn't care about.